.changeset/gold-buses-tap.md

```diff
@@ -0,0 +1,12 @@
+---
+"@biomejs/biome": patch
+---
+
+Fix [#4982](https://github.com/biomejs/biome/issues/4982), js parser should throw a syntax error for `TsImportType` when syntax `import` without arguments after.
+
+The follwing code will throw a syntax error:
+
+```ts
```

Comment:
``````suggestion

Fix [#4982](https://github.com/biomejs/biome/issues/4982), the JavaScript parser now throws a syntax error for the following code:

```ts
``````